lib/envars.ml

```diff
@@ -132,7 +132,8 @@ let guess_coqlib fail =
       if not Coq_config.local && Sys.file_exists (Coq_config.coqlib / prelude)
       then Coq_config.coqlib
       else
-        fail "cannot guess a path for Coq libraries; please use -coqlib option")
+        fail "cannot guess a path for Coq libraries; please use -coqlib option \
+              or instruct Coq not to load the stdlib by using -boot -noinit parameters.")
```

Comment:
This is not user-friendly enough. The message should contain an additional hint like "If you don't understand what this means, it probably means that you installed the wrong package (coq-core instead of coq)."

Reply:
New version.